utilities/cli/container.py

```diff
@@ -381,6 +387,7 @@ def build(
         img: Optional[str] = None,
         no_cache: bool = False,
         build_args: Optional[str] = None,
+        build_scripts: Optional[List[str]] = None,
```

Comment:
I don't understand that interface. Now we have to call `build --build_scripts lint` or `build --build_scripts benchmark` ? That seems not very user-friendly and not something we discussed on the call?